libsel4/arch_include/riscv/interfaces/sel4arch.xml

```diff
@@ -62,6 +62,29 @@
                 in <autoref label="ch:vspace"/>.</docref>
             </description>
             </param>
+            <error name="seL4_DeleteFirst">
+                <description>
+                    A page is mapped in <texttt text="vspace"/> at <texttt text="vaddr."/>
```

Comment:
```suggestion
                    A page is mapped in <texttt text="vspace"/> at <texttt text="vaddr"/>.
```

Reply:
This would look much better in the xml; however, when rendered in the manual, this results in some extra whitespace between ```vaddr``` and the punctuation that looks weird. I'm not sure why that happens.